framework/mesh/mesh_continuum/mesh_continuum.h

```diff
@@ -22,7 +22,27 @@ class MeshGenerator;
  */
 class MeshContinuum
 {
+private:
```

Comment:
I can put functions first, but the public, protected, private ordering for variables won't work here... not without a more in-depth refactoring of `MeshContinuum` and the cell handlers. I was hoping to avoid that for this PR.

Reply:
For the time being, it might be best to create an issue and live with the style violation. I think a clean compile is more important.

Reply:
Maybe, when the we put this in code standards, we need to mention that there are valid cases when deviations are allowed...

Reply:
This can almost certainly be fixed to comply with the coding standards by refactoring `MeshContinuum` and the cell handlers. I just don't have a good enough understanding of those pieces of the code to make a quick fix. Since compilation warnings are very visible to new users and developers, I'm trying to squash them as soon as I notice them (although, I think this one has been around for a while).

Reply:
I agree on the logic. We can talk during our next meeting. I think, sooner rather than later, a better understanding of the MC is needed.

Reply:
Although member variable ordering is not yet in our coding standards, this fix will violate our intended ordering (public, protected, private). We will make an exception in this case for the time being, but a more comprehensive refactoring of `MeshContinuum` needs to be undertaken so this issue can resolved correctly.

Reply:
@wdhawkins please create an issue for this. thx.